### ISSUE TITLE
Interaction of chronic and acute stress factors on coastal ecosystems

### ISSUE THREAD

**Issue by aebudden** [NCEAS/oss-2017#31]
Author: **Vanessa Tobias**
Topics: Restoration

## Summary
Coastal wetlands such as marshes and mangroves provide ecosystem services, including storm protection, food production, water quality improvement, and recreation. Loss of coastal wetlands is a concern in the Gulf Coast; estimates of land loss rates range from 66 to 90 km2 per year. Coastal ecosystems along the Gulf of Mexico face many stressors. Some are chronic stressors such as increased salinity and flooding that result from sea level rise and soil subsidence. Others are acute or episodic stressors such as physical disturbance from hurricanes and the introduction of toxic substances from oil exposure.  Recent papers have shown that acute stress caused by oil exposure increases erosion damage by hurricanes. Understanding how the interaction of chronic and acute stress factors affects coastal ecosystems is critical for planning, but currently the understanding of such interactions on a landscape scale is limited. 

This study will combine data from state, federal, and local sources to describe the role of acute stressors in accelerating land loss due to chronic stressors.  Specifically, this study asks whether and how acute stressors such as oil exposure interact with chronic stressors to increase degradation as measured by various metrics such as land loss rates, vegetation productivity, elevation change, fragmentation at a landscape scale. Recent studies have examined impacts on basin scales, but understanding the wider impacts of environmental stressors is a key factor in developing restoration tools and coastal policy. Restoration projects often focus on mitigating the effects of one or a few chronic sources of stress on coastal wetlands. For example: freshwater diversions reduce salinities and increase nutrient availability in marshes and thin layer placement counteracts flooding effects. Environmental stress is a measurable change of a state that is induced by an environmental change, and that renders the system more vulnerable to further environmental change. The proposed study focuses on the symptoms of stress on coastal wetlands that lead to measurable increases in vulnerability at a landscape scale.

## Data needs
Time series of maps of vegetation types and land loss (lacoast.gov/crms2) indicate where chronic stressors such as salinity and flooding stress vegetation.  Data on water levels, salinity, and soil properties can also provide information about chronic stressors (lacoast.gov/crms2). Locations of oil spills and cleanup efforts (dwhdiver.orr.noaa.gov/) as well as hurricane storm tracks (nhc.noaa.gov) provide information about acute stressors. Published papers and/or data on hunting and fishing could be used to indicate shifts in habitat usage by species of economic concern. Additional data sources will likely be identified during the project, such as those available from the Louisiana Department of Natural Resources (www.sonris.com/).

## Analytical approaches
First, we will develop a conceptual model of how stressors impact coastal ecosystems. This will help identify additional data sources that are needed and suggest hypotheses to test with the data that are available.  The conceptual model will incorporate information from the published literature to define connections and determine which hypotheses have previously been tested. Second, the hypotheses will determine which analytical methods we will use.  Developing a state-space model is a potential method for describing transitions from one habitat to another (such as from salt marsh to open water). Another potential method is to use a functional data analysis approach to look for change points in time series of habitat area to identify when major shifts occurred.  The timing of such shifts could be used to identify potential tipping points in habitat stability. This project will also require techniques for spatial data handling.

## Impact/significance
A comprehensive conceptual model of wetland dynamics as a function of chronic and acute stresses can be used to forecast availability of habitat and ecosystem services under various climate and stress scenarios. These forecasts will help to formulate and evaluate policies to protect wetlands and their associated services. For example, the model could be used as a decision support tool for developing coastal policies by forecasting how habitat availability and their related ecosystem services would change based on decisions.

**Comment by vdtobias** [comment]
The NOAA website on oil spill restoration in the Gulf has moved. Here is the new address: http://www.gulfspillrestoration.noaa.gov/science-data

**Comment by CourtneyPageTan** [comment]
This pairs nicely with my proposal #29! This analysis will aid in the enterprise of identifying physically vulnerable communities. We can identify communities dual threats by overlaying geospatial data on social indicators, such as crime, civic behavior and philanthropic tendencies. The literature has shown that communities with a strong social infrastructure are better prepared to respond threats of climate change and other disturbances. The policy implications of this research are important to promoting policies of building resilience in the Gulf Coast among communities that are facing the threat of climate change, but lack the social infrastructure to engage in collective action to successfully overcome the challenges of the future.

**Comment by vdtobias** [comment]
I agree. There is so much feedback between the physical and social aspects of coastal change, especially around the Gulf.  I like the idea of putting together a project that addresses both.  Proposal #11 might also tie in well with these two.

**Comment by CourtneyPageTan** [comment]
Totally agree, vdtobias! I think #29 #11 and #31 could be combined into a very compelling research project with important policy implications!

**Comment by aesacco** [comment]
This proposal attempts to answer some important questions. One thing you may want to consider is determining a proper spatial scale at which to investigate these changes during your analyses. Underlying environmental processes or signals may be more apparent at more fine or coarse spatial scales, especially in detection of ecosystem shifts that you mentioned.

**Comment by vdtobias** [comment]
I totally agree- the spatial scale is going to be very important.  The conceptual model stage is a good place to consider the spatial scale of the potential driving factors and of the response.  Then you can compare it to the data you actually have and see what is really feasible.
